sha.opam

```diff
@@ -10,10 +10,8 @@ bug-reports:  "https://github.com/djs55/ocaml-sha/issues"
 dev-repo:     "https://github.com/djs55/ocaml-sha.git"
 license:      "ISC"
 
-build: make
-install: [make "install"]
-remove: [["ocamlfind" "remove" "sha"]]
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
```

Comment:
Would be good to have a `build-test` here too (`build-test: ["jbuilder" "runtest" "-p" name]` I think).
Then the CI will run the tests.